hbase/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM anapsix/alpine-java:jdk8
```

Comment:
Let's prefix the directory with the actual hbase version so we could support multiple versions in parallel (We'll have 0.98, 1.x and 2.x likely)